Gemfile

```
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :release do
```

Thoughts on not installing this by default?
```suggestion
group :release, optional: true do
```

It does mean we need to update your release workflow to set `BUNDLE_WITH` but it makes the initial install faster.

---

Doesn't `optional: false` make more sense here? We only need it locally for generating the changelog, but not in general so it could default to false?

---

https://bundler.io/man/gemfile.5.html#BLOCK-FORM-OF-SOURCE-GIT-PATH-GROUP-and-PLATFORMS
> In the case of the group block form the :optional option can be given to prevent a group from being installed unless listed in the --with option given to the bundle install command.

So I think `optional: true` can be read as `default: false`.